app/views/users/profile.html.erb

```diff
@@ -233,12 +233,17 @@
       </div>
     </div>
   <% end %>
+  
```

Comment:
Please add a `<br />` here to add some space between buttons. And, please remove the extra spaces like line #239, 243, 245, 256. Thanks!

Reply:
@uzorjchibuzor the functionality is working great, please do the above changes to optimize design a little. Also, if you have idea for more clean design than you can try.

Thanks!

Reply:
I will implement the changes suggested and also thoroughly look at the code to what can be refactored